test/support/page_objects.js

```diff
@@ -0,0 +1,74 @@
+
+import Common from './pages/common.js';
+import ConsolePage from './pages/console_page.js';
+import DashboardPage from './pages/dashboard_page.js';
+import DiscoverPage from './pages/discover_page.js';
+import HeaderPage from './pages/header_page.js';
+import SettingsPage from './pages/settings_page.js';
+import ShieldPage from './pages/shield_page.js';
+import VisualizePage from './pages/visualize_page.js';
+
+const common = new Common();
+const consolePage = new ConsolePage();
+const dashboardPage = new DashboardPage();
+const discoverPage = new DiscoverPage();
+const headerPage = new HeaderPage();
+const settingsPage = new SettingsPage();
+const shieldPage = new ShieldPage();
+const visualizePage = new VisualizePage();
+
+export default {
```

Comment:
I think that totally makes sense. I'm a little curious about why the context of `this` is unclear though? Is it because I didn't define `remote` as `undefined` on the object?

From my perspective, I see `this` and I assume that (9 times out of 10) it refers to the object on which the method is defined, regardless of whether it's a class prototype object or an object literal. (And in this case, I think of it as a method instead of function because it's a member of the object instead of a function declaration or expression -- sorry if I'm over-explaining myself here).

Considering that we want to expose `remote`, the alternative without using `this` would be to do this:

```javascript
let _remote;

export default {
  get remote() {
    return _remote;
  },

  init(remote) {
    _remote = remote;
    /* ... */
  }

  /* ... */
}
```

I feel like it's jumping through _a couple_ hoops. Though it does protect against someone setting `remote` directly (which is not a big no-no, but is unanticipated), so I'm OK with changing to this pattern, if you still feel it's worth it?